task3/UI/scripts/feedEvents.js

```diff
@@ -0,0 +1,458 @@
+let modal = document.querySelector('.modal');
```

Comment:
Just wondering: why not class?

Reply:
in the beginning it was a small file with a couple of functions ... )